documents/operating.md

```diff
@@ -519,6 +519,17 @@ Here's a summary table:
 | ~ corporate       | Majority           | GitHub       |
 | + minutes         | Apathetic Support  | GitHub       |
 
+### Remote Voting
+Calls to vote made during a plotting session can be voted on remotely by Corporate Overlords.  This is because time is not linear, it is only perceived linearly, therefore all moments of time are actually identical.
+
+Only Corporate Overlords with an active (non hibernating) status at the time of the initial plotting session are able to vote, and the totals needed for a given vote are based on the active corporate overlords at that time.
```

Comment:
I think something that might come up as a question is if a Corporate Overlord misses four plotting sessions in a row but has voted remotely, does that still put the Corporate Overlord into hibernation, even though they've been participating remotely? I would argue that a strict interpretation is that the Corporate Overlord still enters hibernation because forms of participation that are not attendance at plotting sessions are not used to determine hibernation status. If this is the intent, do we need to clarify this?